lib/vrl/stdlib/src/parse_nginx_log.rs

```diff
@@ -204,9 +204,12 @@ fn kind_error() -> BTreeMap<Field, Kind> {
         ("tid".into(), Kind::integer()),
         ("cid".into(), Kind::integer()),
         ("message".into(), Kind::bytes()),
+        ("excess".into(), Kind::float().or_null()),
+        ("zone".into(), Kind::bytes().or_null()),
         ("client".into(), Kind::bytes().or_null()),
         ("server".into(), Kind::bytes().or_null()),
         ("request".into(), Kind::bytes().or_null()),
+        ("upstream".into(), Kind::bytes().or_null()),
```

Comment:
I would have expected `upstream` to be on the other log formats as well, is it not by default? (I may be remembering nginx-ingress)

Reply:
It is not a big deal to add it to the scope of this PR. Should I go with it?

Reply:
No, I think it's fine 👍

Reply:
Another note that I _love_ NGINX formats 😆 

Thanks again for the thoroughness @nabokihms!